src/utilities/customNumberFormat.ts

```diff
@@ -0,0 +1,27 @@
+
+    /**
+     *  Replace '.'  with ','  
```

Comment:
Should this method be used with decimal numbers? Or just integers?
If the numberFormatFraction method should be used for decimal numbers instead, perhaps the comment should not state that it is replacing "." with ","?

Reply:
The only difference between numberFormat and numberFormatFraction is the specified number of fraction digits. So in cases where it's not necessary to show "zeros" behind the comma you can use numberFormat. The idea is that numbers in a column looks better (easier to compare numbers) with trailing zeros.